Docs/source/developers/checksum.rst

```diff
@@ -22,11 +22,6 @@ This relies on the function ``evaluate_checksum``:
 
 .. autofunction:: checksumAPI.evaluate_checksum
 
-Here's an example:
-
-.. literalinclude:: ../../../Examples/Tests/embedded_circle/analysis.py
-   :language: python
-
```

Comment:
This code example is not relevant since #5456 has been merged (in fact, the file does not exist anymore). This checksum section of the documentation will be refactored soon anyways.